session/bootstrap.go

```diff
@@ -64,6 +64,8 @@ const (
 		Create_tmp_table_priv	ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Lock_tables_priv		ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Execute_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Repl_slave_priv	    	ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Repl_client_priv		ENUM('N','Y') NOT NULL DEFAULT 'N',
```

Comment:
The column should be added to the end of the columns.

Reply:
done in https://github.com/pingcap/tidb/pull/21856/commits/b6e1a91a6547476388a8f6416061b741872dece1